guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb

```diff
@@ -18,7 +18,7 @@
    "cell_type": "markdown",
```

Comment:
<ol><li>producing highly accurate<s> x,y,z</s> <strong>X, Y and Z</strong> measurements</li><li>which <s>include x, y, and z,</s> <strong>includes X, Y, and Z measurements</strong> along with additional attributes such as GPS <s>time stamps,</s> <strong>timestamps,</strong></li><li>intensity –&nbsp;the return strength of the laser pulse that generated the Lidar point, <s>and, number of returns – the total number of returns for a given pulse</s> <strong>and the total number of returns for a given pulse.</strong></li><li>Additionally, with further post<strong>-</strong>processing</li><li>Though the <s>PoinCNN</s> <strong>PointCNN </strong>network emulates traditional convolution neural networks</li><li><s>Point Cloud</s> Point cloud</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
''corrected accordingly'', Thanks.

Reply:
<p>You can consider including:</p><p>1) Point clouds are large collections of 3D elevation points, which include x, y, and z <strong>measurements</strong></p><p>2) <s>post -processing</s> <strong>post-processing</strong>  (extra space in between)</p>

Reply:
no extra spaces, were found, in HTML or notebook, might be due to ReviewNB.